Chiazokam/floof
# Implement Get All Banks

#### What does this PR do?
This PR implements Get All Banks API

---

#### Description of Task to be completed?
- implement Response handler
- Implement Error Handler with logger
- implement get all banks
- get data from cache
- implement tryCatch function to wrap controller

---

#### How should this be manually tested?

- Clone this repo
- cd into `implement-get-all-banks-02`
- run npm run start:dev
- check the docs for API samples

---

#### Any background context you want to provide?

N/A

---



#### Screenshots (if appropriate)

N/A

---

#### Questions:

N/A

---
